src/unbound.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+#
+# igcollect - Stats for Unbound DNS cache/resolver
+#
+# Copyright (c) 2016 InnoGames GmbH
+#
+
+from __future__ import print_function
+from argparse import ArgumentParser
+from subprocess import Popen, PIPE
+from time import time
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='unbound')
+    return parser.parse_args()
+
+
+def parse_unbound_stats():
+    out = Popen(('/usr/sbin/unbound-control', 'stats'), stdout=PIPE).\
+        stdout.read().splitlines()
+
+    return {
+        key.replace('total.', ''): val for key, val in (
+            stat.split('=') for stat in out
+        ) if key.startswith('total.')
+    }
```

Comment:
This returns keys under different levels of namespaces.  I don't think it looks good on Graphite.  It would be more useful to replace the dots with underscores.